ansible/roles_ocp_workloads/ocp4_workload_example_dedicated_cluster/tasks/remove_workload.yml

```diff
@@ -13,5 +13,5 @@
 # --------------------------------------------
 - name: remove_workload tasks complete
   debug:
-    msg: "Remove Workload tasks completed successfully."
-  when: not silent|bool
+   msg: "Remove Workload tasks completed successfully."
```

Comment:
Removed space here?

Reply:
yamllint wants indent 3 at that spot. It gives an error if I leave the space.